### PR TITLE
aws cred update

### PR DESCRIPTION
**Pull request by rayashworth**
update to aws cred section.  I left the yaml reference, but noted base64.

### CODE REVIEW

**Review by csrwng — COMMENTED** [review]
Thank you for the PR! just one suggestion

File: docs/using-hive.md [hunk 1]
```diff
@@ -96,8 +96,12 @@ Hive requires credentials to the cloud account into which it will install OpenSh
 
 #### AWS
 
-Create a `secret` containing your AWS access key and secret access key:
+Create a `secret` containing your AWS access key and secret access key:  
 
+```bash
+oc create secret generic <mycluster>-aws-creds -n hive --from-literal=aws_access_key_id=<YOUR_ID> --from-literal=aws_secret_access_key=<YOUR_SECRET>
```
[the remark this answers is not in the document]
Comment (rayashworth) [reply docs/using-hive.md:102]: I agree

[the remark this answers is not in the document]
Comment (rayashworth) [reply docs/using-hive.md]: done

**Comment by gregsheremeta** [comment]
Thanks a lot @rayashworth ! Nice improvement.

**Pull request merged**